packages/rfw/lib/src/flutter/material_widgets.dart

```diff
@@ -499,6 +500,39 @@ Map<String, LocalWidgetBuilder> get _materialWidgetsDefinitions => <String, Loca
     );
   },
 
+  'Slider': (BuildContext context, DataSource source) {
+    // not implemented: overlayColor, mouseCursor, semanticFormatterCallback, focusNode, autofocus
+    final min = source.v<double>(['min']) ?? 0.0;
+    final value = source.v<double>(['value']) ?? min;
+    final labelText = source.v<String>(['label']);
+    final label = labelText != null ? '$labelText:${value.toStringAsFixed(2)}' : value.toStringAsFixed(2);
+    return Slider(
+      value: value,
+      secondaryTrackValue: source.v<double>(['secondaryTrackValue']),
+      onChanged: source.handler(['onChanged'],
+              (HandlerTrigger trigger) => (double value) {
+            trigger({'value': value});
+          }),
```

Comment:
Per the [style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo), indentation should never have a line that is less indented than a line that is of higher lexical scope (well the style guide doesn't say it quite that way but that's the intent).
```suggestion
      onChanged: source.handler(['onChanged'],
        (HandlerTrigger trigger) => (double value) {
          trigger({'value': value});
        },
      ),
```

Reply:
(same below)